sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveDataFrameAnalyticsSuite.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.sql.hive
```

Comment:
The unit tests for the DataFrame methods should be in the base sql package not hive so that they are run when not compiled with hive support.